appserver/appclient/server/core/src/main/java/org/glassfish/appclient/server/core/jws/RestrictedContentAdapter.java

```diff
@@ -379,9 +379,6 @@ protected void finishErrorResponse(final Response gResp, final int status) {
     }
 
     protected String dumpContent() {
-        if (content == null) {
```

Comment:
Content is a final field initialised in the constructor. So unless there is some initialisation madness going on sonar should be correct that this cannot be null.